parser_test.go

```diff
@@ -4773,63 +4773,6 @@ func TestLimits(t *testing.T) {
 }
 
 func TestDisallowSubqueriesOnStatements(t *testing.T) {
```

Comment:
instead of removing these tests, would we want to just validate the statements _do_ work?

Reply:
Yeah sure